### ISSUE TITLE
Cosmos.imports.ts isn't created nor updated on subsequent runs of npm run cosmos

### ISSUE THREAD

**Issue by Inukares** [react-cosmos/react-cosmos#1549]
### What's wrong?

Imports seem to be generated only once and afterwards react-cosmos server starts without regenerating them.
That means, that If I add other fixtures they don't seem to be added to cosmos.imports file, or if cosmos.imports file has been deleted accidentally there's no easy way to recover ( as it is recommended to add it to gitignore ).

### Steps to reproduce

1. Follow guide on setup with Next.js
2.  Run npm run cosmos
3.  Add another fixture on top of existing ones
4. The cosmos.imports file doesn't mention the newly added fixture. Re-runing `npm run cosmos` doesn't do the trick either.

### Screenshots

Simply no 'generated cosmos.imports.ts' prompt is visible in the console

### Useful info

- Cosmos version.
6.0.0-beta.6 of both react-cosmos and react-cosmos-next
- Cosmos config.
{
  port: 5001,
  "rendererUrl": {
     "dev" : "http://localhost:3000/cosmos/<fixture>",
     "export": "/cosmos/<fixture>.html"
  }
}
- Server terminal output.
```
[Cosmos] Using cosmos config found at cosmos.config.json
[Cosmos] See you at http://localhost:5001
```
*notice no output about generated config*
- Browser console output.
`Cannot GET /[object%20Object]` on fixture clicking 
`Failed to load resource: the server responded with a status of 404 (Not Found)`
( I assume that's because I don't  currently have cosmos.imports.ts file, because it used to work with exact same config and when the file was there )

### Additional context

I want to be able to add new fixtures and access them on the web. I'd expect that cosmos.imports file gets overwritten whenever something happens, but that doesn't seem to be the case.
I may be missing something, but this behaviour seems very confusing. I had no trouble setting up react-cosmos with nextjs in the first place, but this issue stops me from progressing with the library.

I found in the codebase pieces about fixture cache and that it is used in context of renderer, but not sure if it is relevant in any way.
I also found `exposeImports` is used in the code which should be taken from config file so I tried adding `"exposeImports": true` but to no avail.

Am I missing something?

**Comment by skidding** [comment]
Hi @Inukares 

Please check out this issue: https://github.com/react-cosmos/react-cosmos/issues/1545

Let me know if it's the same issue related to mismatched package versions.

**Comment by Inukares** [comment]
Don't tell me wife but I might be falling for you. It worked!

**Comment by skidding** [comment]
Yes, that's correct. The '^' part is the issue with beta releases. Hopefully we fix this confusion in the future but for now exact versions should be used until we release 6.0 official.
